src/apps/frontend/types/account.ts

```diff
@@ -14,7 +14,7 @@ export class Account {
     this.lastName = json.last_name as string;
     this.phoneNumber = json.phone_number
       ? new PhoneNumber(json.phone_number as JsonObject)
-      : null;
+      : undefined;
```

Comment:
At some places, we are defaulting to null, and some places we are defaulting to `undefined`.

It seems we need to adopt a standard practice to use either null or undefined across the code base to make absence of data. `undefined` is preferred.